resp/response.go

```diff
@@ -1,6 +1,6 @@
 package resp
 
-//go:generate bash -c "go run generate/*.go | gofmt -s > auto.go"
+//go:generate bash -c "go run generate/*.go | gofmt -s > auto.go; test -s auto.go || rm auto.go"
```

Comment:
This sounds picky, but rather than testing after the fact and removing, why not simply remove before the fact or just set bash to clobber the file, instead?

I suspect that will ultimately be easier to understand for a wider audience.

Reply:
Ah, yeah, ignore that - I just figured out why :+1: